content/en/docs/setup/_index.md

```diff
@@ -103,6 +103,7 @@ The following production environment solutions table lists the providers and the
 | [Pivotal](https://pivotal.io/) | | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service) | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service)           |      |          |
 | [Platform9](https://platform9.com/)         | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)             |       | [Platform9 Managed Kubernetes](https://platform9.com/managed-kubernetes/)            |  &#x2714;    |  &#x2714;    |   &#x2714;
 | [Rancher](https://rancher.com/)         |              |   [Rancher 2.x](https://rancher.com/docs/rancher/v2.x/en/)     |             |  [Rancher Kubernetes Engine (RKE)](https://rancher.com/docs/rke/latest/en/)    |          | [k3s](https://k3s.io/)
+| [Scaleway](https://www.scaleway.com/)        | [Scaleway Kapsule (Beta)](https://www.scaleway.com/en/kubernetes/)    | [Scaleway Instances](https://www.scaleway.com/en/virtual-instances/) |      |     |            |
```

Comment:
Would it be OK to hold off until the Kapsule service has left public beta?
(My opinion): readers looking for a turnkey solution would want a list of stable services.